.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Summary
+
+<!-- What is this pull request for? Does it fix any issues? -->
+
+### Checklist
+
+- [ ] If code changes were made then they have been tested.
+    - [ ] I have updated the documentation to reflect the changes.
```

Comment:
Should there be a checkbox about code style/linting?

Reply:
I think if this were to be added then I would need to also add a CONTRIBUTING.md file.